Source/RomanNumeral.c

```diff
@@ -10,7 +10,144 @@ static char * Append(char *output, const char *toAppend)
 {
    return output + sprintf(output, "%s", toAppend);
 }
-
 void RomanNumeral_Convert(unsigned input, char *output)
 {
+   int i, hundreds, tens, units;
+   i = 0;
+   hundreds = (input / 100) % 10;
+   tens = (input - (hundreds*100)) / 10;
+   units = (input - (hundreds*100 + tens*10));
+   for(i = 0; i <=100; i++)
+      {
+         output[i] = '\0';
+      }
+   switch (hundreds)
+      {
+      case 1: 
+         *output = 'C';
+         output++;
+         break;
+      default: 
+         break;
+      }
+
+   switch (tens)
+      {
+         case 1:
+            *output = 'X';
+            output++;
+            break;
+         case 2:
+            *output = 'X';
+            output++;
+            *output = 'X';
+            output++;
+            break;
+         case 3:
+            *output = 'X';
+            output++;
+            *output = 'X';
+            output++;
+            *output = 'X';
+            output++;
+            break;
+         case 4:
+            *output = 'X';
+            output++;
+            *output = 'L';
+            output++;
+            break;
+         case 5: printf ("L"); 
+            *output = 'L';
+            output++;
+            break;
+         case 6: 
+            *output = 'L';
+            output++;
+            *output = 'X';
+            output++;
+            break;
+         case 7:
+            *output = 'L';
+            output++;
+            *output = 'X';
+            output++;
+            *output = 'X';
+            output++;
+            break;
+         case 8:
+            *output = 'L';
+            output++;
+            *output = 'X';
+            output++;
+            *output = 'X';
+            output++;
+            *output = 'X';
+            output++;
+            break;
+         case 9:
+            *output = 'X';
+            output++;
+            *output = 'C';
+            output++;
+            break;
+
+         default: 
+            break; 
+      }
+
+   switch (units) 
```

Comment:
any way you can convine your tents and units switch cases? to me look very similar and we can simplify there.